bin/sdk/gutenberg.js

```diff
@@ -95,6 +104,15 @@ exports.config = ( { argv: { inputDir, outputDir }, getBaseConfig } ) => {
 						betaBlocks: presetBetaBlocks,
 					},
 				} ),
+			new webpack.BannerPlugin( {
+				banner: compact( [
+					new Date().toUTCString(),
+					// Commit and Calypso repository URL are added only when
+					// inputDir is within Calypso directory
+					commit && `Commit: ${ commit }`,
+					inputDirIsInCalypso && 'Repository: https://github.com/Automattic/wp-calypso/',
```

Comment:
I considered adding `--banner` argument to CLI but I don't think we really need the flexibility.